script.js

```diff
@@ -0,0 +1,402 @@
+openlinkContact.addEventListener('click', () => {
+  document.getElementById('side-bar').style.display = 'none';
+  window.location.reload();
+});
+
+//  popup window mobile//
+
+const popUpblack = document.createElement('div');
+popUpblack.id = 'popupid';
+popUpblack.className = 'popupclass';
+document.body.appendChild(popUpblack);
+
+const popUp = document.createElement('div');
+popUp.id = 'content';
+popUp.className = 'note';
+popUpblack.appendChild(popUp);
+
+const popTextdiv = document.createElement('div');
+popTextdiv.id = 'poptest';
+popTextdiv.className = 'poptesty';
+popUp.appendChild(popTextdiv);
+
+const popupTitle = document.createElement('h3');
+popupTitle.textContent = 'Multi Post Stories';
+popupTitle.id = 'popuptitleid';
+popupTitle.className = 'poptitleclass';
+popTextdiv.appendChild(popupTitle);
+
+const closeicon = document.createElement('img');
+closeicon.src = 'image/close.svg';
+closeicon.id = 'closeiconid';
+closeicon.className = 'closeiconclass';
+popTextdiv.appendChild(closeicon);
+
+const imageTextblock = document.createElement('div');
+imageTextblock.id = 'imagetxtid';
+imageTextblock.className = 'imagetxtclass';
+popUp.appendChild(imageTextblock);
+
+const list = document.createElement('ul');
+list.id = 'listing';
+list.className = 'listing';
+imageTextblock.appendChild(list);
+
+const listchild1 = document.createElement('li');
+listchild1.textContent = 'html';
+listchild1.id = 'bam';
+listchild1.className = 'bambam';
+list.appendChild(listchild1);
+
+const listchild2 = document.createElement('li');
+listchild2.textContent = 'bootstrap';
+listchild2.id = 'blam';
+listchild2.className = 'banban';
+list.appendChild(listchild2);
+
+const listchild3 = document.createElement('li');
+listchild3.textContent = 'Ruby on rails';
+listchild3.id = 'lone';
+listchild3.className = 'lome';
+list.appendChild(listchild3);
+
+const image = document.createElement('img');
+image.src = 'image/mobile-plain.png';
+image.id = 'placement';
+image.className = 'placemenImage';
+imageTextblock.appendChild(image);
+
+const popupText = document.createElement('p');
+popupText.textContent = "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent";
+popupText.id = 'ipsum';
+popupText.className = 'lorem';
+imageTextblock.appendChild(popupText);
+
+const buttonblock = document.createElement('div');
+buttonblock.id = 'blank';
+buttonblock.className = 'create';
+popUp.appendChild(buttonblock);
+
+const buttonone = document.createElement('button');
+buttonone.textContent = 'See Live';
+buttonone.id = 'orange';
+buttonone.className = 'orang';
+buttonblock.appendChild(buttonone);
+
+const icon = document.createElement('img');
+icon.src = 'image/icon-see.svg';
```

Comment:
- [ ] I think there is an issue with one of the images of the popup window because it's not appearing. Please, kindly fix this issue for a better user experience :smiley:
![image](https://user-images.githubusercontent.com/81522435/147255364-df8765c4-fa19-40f8-84e5-42cd60e1308b.png)

Reply:
Done. Thanks